presto-product-tests/bin/run_on_docker.sh

```diff
@@ -70,25 +70,46 @@ function stop_application_runner_containers() {
   done
 }
 
+function stop_all_containers() {
+  local ENVIRONMENT
+  for ENVIRONMENT in $(getAvailableEnvironments)
+  do
+     stop_docker_compose_containers
+  done
+}
+
 function stop_docker_compose_containers() {
-  RUNNING_CONTAINERS=$(docker-compose -f "$1" ps -q)
+  RUNNING_CONTAINERS=$(environment_docker_compose ps -q)
 
   if [[ ! -z ${RUNNING_CONTAINERS} ]]; then
     # stop application runner containers started with "run"
-    stop_application_runner_containers "$1"
+    stop_application_runner_containers
 
     # stop containers started with "up"
-    docker-compose -f "$1" down
+    environment_docker_compose down
   fi
 
   echo "Docker compose containers stopped: [$1]"
 }
 
+function environment_docker_compose() {
+  local ENVIRONMENT_DOCKER_COMPOSE_LOCATION="${DOCKER_CONF_LOCATION}/${ENVIRONMENT}/docker-compose.yml"
+  local COMPOSE_FILES=( -f "${DOCKER_CONF_LOCATION}/common/standard.yml" )
+  if [[ $ENVIRONMENT == *"kerberos"* ]]; then
+    COMPOSE_FILES+=( -f "${DOCKER_CONF_LOCATION}/common/kerberos.yml" )
+  fi
+  COMPOSE_FILES+=( \
+    -f "${DOCKER_CONF_LOCATION}/common/jdbc_db.yml" \
+    -f "${ENVIRONMENT_DOCKER_COMPOSE_LOCATION}" \
+  )
+  docker-compose "${COMPOSE_FILES[@]}" "$@"
+}
```

Comment:
maybe instead of this function, you could have a simple script in environment directory which would have specific for this environment docker-compose command. This give more copy-paste but
 - gives user possibility to run environment docker cluster without much thinking 
 - user can easy check the configuration (what overridden what) with - https://docs.docker.com/compose/reference/config/
 - it will be easier to run manual tests